<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grabcut_Widget</class>
 <widget class="QWidget" name="Grabcut_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>641</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>GrabCutTool Controls</string>
   </property>
   <widget class="QLabel" name="thickness_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Brush Thickness</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="thickness_spinbox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="color_combobox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Definite Background</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Definite Foreground</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Probable Background</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Probable Foreground</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="color_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="done_pushbtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save and Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iter_pushbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterate Grabcut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_pushbtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_pushbtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QSlider" name="transparency_slider">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mask Transparency</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="editor_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
